### ISSUE TITLE
Problems occurred when invoking code from plug-in: "org.eclipse.jface".

### ISSUE THREAD

**Issue by Vlczech** [dbeaver/dbeaver#8086]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version: DBeaver 7.0
- Additional extensions: -

#### Connection specification:
- Database name and version: PostgreSQL 11
- Driver name: PostgreSQL JDBC driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Plugin: `org.eclipse.jface`
Stack Trace:
```
java.lang.IllegalStateException: No databases found on the server
	at org.jkiss.dbeaver.ext.postgresql.model.PostgreDataSource.getDefaultInstance(PostgreDataSource.java:388)
	at org.jkiss.dbeaver.ext.postgresql.model.PostgreDataSource.getDefaultInstance(PostgreDataSource.java:1)
	at org.jkiss.dbeaver.model.DBUtils.getObjectOwnerInstance(DBUtils.java:1897)
	at org.jkiss.dbeaver.model.DBUtils.getDefaultContext(DBUtils.java:1904)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.isDefaultElement(DBNUtils.java:129)
	at org.jkiss.dbeaver.ui.navigator.database.DatabaseNavigatorLabelProvider.getFont(DatabaseNavigatorLabelProvider.java:120)
	at org.eclipse.jface.viewers.ColumnLabelProvider.update(ColumnLabelProvider.java:42)
	at org.eclipse.jface.viewers.ViewerColumn.refresh(ViewerColumn.java:144)
	at org.eclipse.jface.viewers.AbstractTreeViewer.doUpdateItem(AbstractTreeViewer.java:946)
	at org.eclipse.jface.viewers.AbstractTreeViewer$UpdateItemSafeRunnable.run(AbstractTreeViewer.java:121)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.ui.internal.JFaceUtil.lambda$0(JFaceUtil.java:47)
	at org.eclipse.jface.util.SafeRunnable.run(SafeRunnable.java:174)
	at org.eclipse.jface.viewers.AbstractTreeViewer.doUpdateItem(AbstractTreeViewer.java:1024)
	at org.eclipse.jface.viewers.StructuredViewer$UpdateItemSafeRunnable.run(StructuredViewer.java:422)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.ui.internal.JFaceUtil.lambda$0(JFaceUtil.java:47)
	at org.eclipse.jface.util.SafeRunnable.run(SafeRunnable.java:174)
	at org.eclipse.jface.viewers.StructuredViewer.updateItem(StructuredViewer.java:2102)
	at org.eclipse.jface.viewers.AbstractTreeViewer.updateChildren(AbstractTreeViewer.java:2770)
	at org.eclipse.jface.viewers.AbstractTreeViewer.internalRefreshStruct(AbstractTreeViewer.java:1951)
	at org.eclipse.jface.viewers.TreeViewer.internalRefreshStruct(TreeViewer.java:677)
	at org.eclipse.jface.viewers.AbstractTreeViewer.internalRefreshStruct(AbstractTreeViewer.java:1957)
	at org.eclipse.jface.viewers.TreeViewer.internalRefreshStruct(TreeViewer.java:677)
	at org.eclipse.jface.viewers.AbstractTreeViewer.internalRefresh(AbstractTreeViewer.java:1927)
	at org.eclipse.jface.viewers.AbstractTreeViewer.internalRefresh(AbstractTreeViewer.java:1884)
	at org.eclipse.jface.viewers.StructuredViewer.lambda$3(StructuredViewer.java:1477)
	at org.eclipse.jface.viewers.StructuredViewer.preservingSelection(StructuredViewer.java:1393)
	at org.eclipse.jface.viewers.TreeViewer.preservingSelection(TreeViewer.java:363)
	at org.eclipse.jface.viewers.StructuredViewer.preservingSelection(StructuredViewer.java:1354)
	at org.eclipse.jface.viewers.StructuredViewer.refresh(StructuredViewer.java:1477)
	at org.eclipse.jface.viewers.ColumnViewer.refresh(ColumnViewer.java:538)
	at org.jkiss.dbeaver.ui.navigator.database.DatabaseNavigatorTree.nodeChanged(DatabaseNavigatorTree.java:280)
	at org.jkiss.dbeaver.model.navigator.DBNModel$EventProcessingJob.lambda$0(DBNModel.java:677)
	at org.jkiss.dbeaver.ui.UIExecutionQueue.executeInUI(UIExecutionQueue.java:52)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:3963)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3590)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1160)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1049)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:660)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:559)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:154)
	at org.jkiss.dbeaver.core.application.DBeaverApplication.start(DBeaverApplication.java:233)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:657)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:594)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1465)
```

Session Data:
```
eclipse.buildId=unknown
java.version=11.0.5
java.vendor=AdoptOpenJDK
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Command-line arguments:  -os win32 -ws win32 -arch x86_64
```

#### Steps to reproduce, if exist:

I think, that it is related to a VPN disconnecting / outage

#### Include any warning/errors/backtraces from the logs

![obrazek](https://user-images.githubusercontent.com/10884140/75879951-a47d6d00-5e1c-11ea-9d81-14106e07f17a.png)

**Comment by Vlczech** [comment]
This appeared again for this scenario after disconnected and again connected VPN: The tree of database navigator is opened to tables. I force reconnect by right click, context menu and selecting item "Invalidate/Reconnect". Then I right click on for example database and select "Refresh" on context menu...

**Comment by uslss** [comment]
closed as duplicate of #8085
